### PR TITLE
feat(core): update nx & angular to 16 version

### PR DESCRIPTION
**Pull request by ZhmenZH**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,54 +38,59 @@
...
+    "jest-preset-angular": "13.1.1",
+    "ng-packagr": "16.0.1",
+    "nx": "16.3.1",
+    "nx-cloud": "16.0.5",
+    "patch-package": "^7.0.0",
     "postcss": "^8.3.9",
     "postcss-import": "14.1.0",
     "postcss-preset-env": "7.5.0",
     "postcss-url": "10.1.3",
+    "postinstall-postinstall": "^2.1.0",
```
Comment (dmitry-stepanenko) [inline package.json:87]: was this added by migrations?

Comment (ZhmenZH) [reply package.json:87]: no, this was added manually for patch-package

Comment (ZhmenZH) [reply package.json:87]: no, this was added manually for patch-package

Comment (dmitry-stepanenko) [reply package.json:87]: ah, that's because of yarn v1, got it

**Review by dmitry-stepanenko — APPROVED** [review]

**Review by Alexveselun — APPROVED** [review]
Testing completed. No critical issues were found

**Pull request merged**